### DISCUSSION TITLE
Log bleeds through above workflow title

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#13667] [Actions] [answered]
If you load this and scroll (repeatedly up and down helps):
https://github.com/check-spelling/hugegraph/runs/5713197458?check_suite_focus=true
<img width="1314" alt="image" src="https://user-images.githubusercontent.com/2119212/160310284-f54ee252-1fc5-4f8c-be0a-6a18b993b5ee.png">

you'll see some white pixels are bleeding through above the block that has the title for the workflow.

<img width="1329" alt="image" src="https://user-images.githubusercontent.com/2119212/160310346-cc807b1f-b89a-40ed-a073-750ecfdc0744.png">

**Comment by chrispat** [comment, marked as answer]
Thank you for the feedback, I have filed an issue for the team.
